.github/workflows/schedule-hourly.yml

```diff
@@ -25,12 +25,13 @@ jobs:
       - name: Run Bazel Build All No Cache
         uses: ./.github/actions/bazel-test-all/
         with:
-          # --config=release is required for the BNs as they rely for both dev and prod deployment
-          # on images being uploaded to the S3 bucket
-          BAZEL_COMMAND: build --config=ci --config=release --repository_cache= --disk_cache= --noremote_accept_cached --remote_instance_name=${CI_COMMIT_SHA} --@rules_rust//rust/settings:pipelined_compilation=True
+          BAZEL_COMMAND: build --config=ci --repository_cache= --disk_cache= --noremote_accept_cached --remote_instance_name=${CI_COMMIT_SHA} --@rules_rust//rust/settings:pipelined_compilation=True
           BAZEL_TARGETS: //...
           CLOUD_CREDENTIALS_CONTENT: ${{ secrets.CLOUD_CREDENTIALS_CONTENT }}
           GPG_PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
+          # 'upload-artifacts' is required for the BNs as they rely for both dev and prod deployment
+          # on images being uploaded to the S3 bucket
+          upload-artifacts: true
```

Comment:
To retain the old behaviour we would also requite:
```
release-build: true
```
right?

Reply:
`release-build: true` is the default!

Reply:
Gotcha!